### ISSUE TITLE
WebpackDevServer doesn't work with awesome-typescript-loader, ExtractTextPlugin, and sass-loader

### ISSUE THREAD

**Issue by fongandrew** [s-panferov/awesome-typescript-loader#36]
Using the dev server with awesome-typescript-loader, ExtractTextPlugin, and sass-loader results in a `callback(): The callback was already called` error. Not sure exactly what the issue is, but the issue is only triggered when all three plugins / loaders are present.

Related issues here:
https://github.com/webpack/extract-text-webpack-plugin/issues/88
https://github.com/jtangelder/sass-loader/issues/125

Complete repro here:
https://github.com/fongandrew/webpack-extract-sass-test/tree/master

**Comment by s-panferov** [comment]
@fongandrew 	hello. I've just cloned the repo and everything builds fine. Maybe you have some deps outdated?

```
➜  webpack-extract-sass-test git:(master) ./node_modules/.bin/webpack
Hash: 404b8b3443c6619b766e
Version: webpack 1.10.3
Time: 1384ms
   [0] ./src/index.js 54 bytes {0} [built]
    + 2 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules
```

```
➜  webpack-extract-sass-test git:(master) ls dist
bundle.css bundle.js
```

```
➜  webpack-extract-sass-test git:(master) npm list
webpack-sass-test@0.1.0 /Users/panferov-s/Workspace/webpack-extract-sass-test
├─┬ awesome-typescript-loader@0.8.0-rc.0
│ ├── bluebird@2.9.34
│ ├── colors@1.1.2
│ ├─┬ loader-utils@0.2.11
│ │ ├── big.js@3.1.3
│ │ └── json5@0.4.0
│ ├── lodash@3.10.0
│ ├── ntypescript@1.201507220720.1
│ ├── object-assign@2.1.1
│ └── typescript@1.5.3
├─┬ css-loader@0.15.5
│ ├─┬ css-selector-tokenizer@0.5.3
│ │ ├── cssesc@0.1.0
│ │ └── fastparse@1.1.1
│ ├─┬ cssnano@1.4.3
│ │ ├── balanced-match@0.2.0
│ │ ├── css-list@0.1.2
│ │ ├── flatten@0.0.1
│ │ ├── indexes-of@1.0.1
│ │ ├── minimist@1.1.2
│ │ ├─┬ postcss-calc@4.1.0
│ │ │ ├── postcss-message-helpers@2.0.0
│ │ │ └─┬ reduce-css-calc@1.2.0
│ │ │   ├── balanced-match@0.1.0
│ │ │   └── reduce-function-call@1.0.1
│ │ ├─┬ postcss-colormin@1.2.3
│ │ │ ├─┬ color@0.8.0
│ │ │ │ ├── color-convert@0.5.3
│ │ │ │ └─┬ color-string@0.3.0
│ │ │ │   └── color-name@1.0.0
│ │ │ ├─┬ colormin@1.0.2
│ │ │ │ ├─┬ color@0.9.0
│ │ │ │ │ ├── color-convert@0.5.3
│ │ │ │ │ └─┬ color-string@0.3.0
│ │ │ │ │   └── color-name@1.0.0
│ │ │ │ └── css-color-names@0.0.1
│ │ │ └─┬ reduce-function-call@1.0.1
│ │ │   └── balanced-match@0.1.0
│ │ ├── postcss-convert-values@1.2.3
│ │ ├─┬ postcss-discard-comments@1.2.0
│ │ │ └── node-balanced@0.0.14
│ │ ├── postcss-discard-duplicates@1.1.3
│ │ ├── postcss-discard-empty@1.1.1
│ │ ├─┬ postcss-discard-unused@1.0.0
│ │ │ └── uniqs@2.0.0
│ │ ├─┬ postcss-font-family@1.1.1
│ │ │ └── uniqs@2.0.0
│ │ ├── postcss-merge-idents@1.0.1
│ │ ├── postcss-merge-rules@1.3.1
│ │ ├── postcss-minify-font-weight@1.0.1
│ │ ├─┬ postcss-minify-selectors@1.4.2
│ │ │ ├── javascript-natural-sort@0.7.1
│ │ │ ├── normalize-selector@0.0.3-a
│ │ │ ├─┬ postcss-selector-parser@1.1.1
│ │ │ │ └── uniq@1.0.1
│ │ │ └── uniqs@2.0.0
│ │ ├─┬ postcss-normalize-url@1.2.1
│ │ │ ├── is-absolute-url@1.0.0
│ │ │ ├─┬ normalize-url@1.3.0
│ │ │ │ ├── object-assign@3.0.0
│ │ │ │ ├── prepend-http@1.0.1
│ │ │ │ ├── query-string@2.3.0
│ │ │ │ └─┬ sort-keys@1.1.0
│ │ │ │   └── is-obj@1.0.0
│ │ │ └── object-assign@2.1.1
│ │ ├── postcss-pseudoelements@2.2.0
│ │ ├─┬ postcss-reduce-idents@1.0.1
│ │ │ └─┬ reduce-function-call@1.0.1
│ │ │   └── balanced-match@0.1.0
│ │ ├── postcss-single-charset@0.2.2
│ │ ├─┬ postcss-unique-selectors@1.0.0
│ │ │ ├── javascript-natural-sort@0.7.1
│ │ │ └── uniqs@2.0.0
│ │ ├── postcss-zindex@1.1.2
│ │ ├─┬ read-file-stdin@0.2.0
│ │ │ └── gather-stream@1.0.0
│ │ └── write-file-stdout@0.0.2
│ ├─┬ loader-utils@0.2.11
│ │ ├── big.js@3.1.3
│ │ └── json5@0.4.0
│ ├─┬ postcss@4.1.16
│ │ ├── es6-promise@2.3.0
│ │ ├── js-base64@2.1.8
│ │ └─┬ source-map@0.4.4
│ │   └── amdefine@1.0.0
│ ├── postcss-modules-extract-imports@0.0.5
│ ├── postcss-modules-local-by-default@0.0.10
│ ├── postcss-modules-scope@0.0.7
│ └── source-list-map@0.1.5
├─┬ extract-text-webpack-plugin@0.8.2
│ ├── async@1.4.0
│ └─┬ loader-utils@0.2.11
│   ├── big.js@3.1.3
│   └── json5@0.4.0
├─┬ node-sass@3.2.0
│ ├── async-foreach@0.1.3
│ ├─┬ chalk@1.1.0
│ │ ├── ansi-styles@2.1.0
│ │ ├── escape-string-regexp@1.0.3
│ │ ├─┬ has-ansi@2.0.0
│ │ │ └── ansi-regex@2.0.0
│ │ ├─┬ strip-ansi@3.0.0
│ │ │ └── ansi-regex@2.0.0
│ │ └── supports-color@2.0.0
│ ├─┬ gaze@0.5.1
│ │ └─┬ globule@0.1.0
│ │   ├─┬ glob@3.1.21
│ │   │ ├── graceful-fs@1.2.3
│ │   │ └── inherits@1.0.0
│ │   ├── lodash@1.0.2
│ │   └─┬ minimatch@0.2.14
│ │     ├── lru-cache@2.6.5
│ │     └── sigmund@1.0.1
│ ├── get-stdin@4.0.1
│ ├─┬ glob@5.0.14
│ │ ├─┬ inflight@1.0.4
│ │ │ └── wrappy@1.0.1
│ │ ├── inherits@2.0.1
│ │ ├─┬ minimatch@2.0.10
│ │ │ └─┬ brace-expansion@1.1.0
│ │ │   ├── balanced-match@0.2.0
│ │ │   └── concat-map@0.0.1
│ │ ├─┬ once@1.3.2
│ │ │ └── wrappy@1.0.1
│ │ └── path-is-absolute@1.0.0
│ ├─┬ meow@3.3.0
│ │ ├─┬ camelcase-keys@1.0.0
│ │ │ ├── camelcase@1.1.0
│ │ │ └── map-obj@1.0.1
│ │ ├─┬ indent-string@1.2.2
│ │ │ └─┬ repeating@1.1.3
│ │ │   └─┬ is-finite@1.0.1
│ │ │     └── number-is-nan@1.0.0
│ │ ├── minimist@1.1.2
│ │ └── object-assign@3.0.0
│ ├─┬ mkdirp@0.5.1
│ │ └── minimist@0.0.8
│ ├── nan@1.8.4
│ ├─┬ npmconf@2.1.2
│ │ ├─┬ config-chain@1.1.9
│ │ │ └── proto-list@1.2.4
│ │ ├── inherits@2.0.1
│ │ ├── ini@1.3.4
│ │ ├─┬ nopt@3.0.3
│ │ │ └── abbrev@1.0.7
│ │ ├─┬ once@1.3.2
│ │ │ └── wrappy@1.0.1
│ │ ├─┬ osenv@0.1.3
│ │ │ ├── os-homedir@1.0.1
│ │ │ └── os-tmpdir@1.0.1
│ │ ├── semver@4.3.6
│ │ └── uid-number@0.0.5
│ ├─┬ pangyp@2.2.1
│ │ ├─┬ fstream@1.0.7
│ │ │ └── inherits@2.0.1
│ │ ├─┬ glob@4.3.5
│ │ │ ├─┬ inflight@1.0.4
│ │ │ │ └── wrappy@1.0.1
│ │ │ ├── inherits@2.0.1
│ │ │ └─┬ once@1.3.2
│ │ │   └── wrappy@1.0.1
│ │ ├── graceful-fs@3.0.8
│ │ ├─┬ minimatch@2.0.10
│ │ │ └─┬ brace-expansion@1.1.0
│ │ │   ├── balanced-match@0.2.0
│ │ │   └── concat-map@0.0.1
│ │ ├─┬ nopt@3.0.3
│ │ │ └── abbrev@1.0.7
│ │ ├─┬ npmlog@1.0.0
│ │ │ ├── ansi@0.3.0
│ │ │ ├─┬ are-we-there-yet@1.0.4
│ │ │ │ ├── delegates@0.1.0
│ │ │ │ └─┬ readable-stream@1.1.13
│ │ │ │   ├── core-util-is@1.0.1
│ │ │ │   ├── inherits@2.0.1
│ │ │ │   ├── isarray@0.0.1
│ │ │ │   └── string_decoder@0.10.31
│ │ │ └─┬ gauge@1.0.2
│ │ │   └── has-unicode@1.0.0
│ │ ├─┬ osenv@0.1.3
│ │ │ ├── os-homedir@1.0.1
│ │ │ └── os-tmpdir@1.0.1
│ │ ├─┬ request@2.51.0
│ │ │ ├── aws-sign2@0.5.0
│ │ │ ├─┬ bl@0.9.4
│ │ │ │ └─┬ readable-stream@1.0.33
│ │ │ │   ├── core-util-is@1.0.1
│ │ │ │   ├── inherits@2.0.1
│ │ │ │   ├── isarray@0.0.1
│ │ │ │   └── string_decoder@0.10.31
│ │ │ ├── caseless@0.8.0
│ │ │ ├─┬ combined-stream@0.0.7
│ │ │ │ └── delayed-stream@0.0.5
│ │ │ ├── forever-agent@0.5.2
│ │ │ ├─┬ form-data@0.2.0
│ │ │ │ ├── async@0.9.2
│ │ │ │ └─┬ mime-types@2.0.14
│ │ │ │   └── mime-db@1.12.0
│ │ │ ├─┬ hawk@1.1.1
│ │ │ │ ├── boom@0.4.2
│ │ │ │ ├── cryptiles@0.2.2
│ │ │ │ ├── hoek@0.9.1
│ │ │ │ └── sntp@0.2.4
│ │ │ ├─┬ http-signature@0.10.1
│ │ │ │ ├── asn1@0.1.11
│ │ │ │ ├── assert-plus@0.1.5
│ │ │ │ └── ctype@0.5.3
│ │ │ ├── json-stringify-safe@5.0.1
│ │ │ ├── mime-types@1.0.2
│ │ │ ├── node-uuid@1.4.3
│ │ │ ├── oauth-sign@0.5.0
│ │ │ ├── qs@2.3.3
│ │ │ ├── stringstream@0.0.4
│ │ │ ├── tough-cookie@2.0.0
│ │ │ └── tunnel-agent@0.4.1
│ │ ├── rimraf@2.2.8
│ │ ├── semver@4.3.6
│ │ ├─┬ tar@1.0.3
│ │ │ ├── block-stream@0.0.8
│ │ │ └── inherits@2.0.1
│ │ └── which@1.0.9
│ ├─┬ request@2.60.0
│ │ ├── aws-sign2@0.5.0
│ │ ├─┬ bl@1.0.0
│ │ │ └─┬ readable-stream@2.0.2
│ │ │   ├── core-util-is@1.0.1
│ │ │   ├── inherits@2.0.1
│ │ │   ├── isarray@0.0.1
│ │ │   ├── process-nextick-args@1.0.2
│ │ │   ├── string_decoder@0.10.31
│ │ │   └── util-deprecate@1.0.1
│ │ ├── caseless@0.11.0
│ │ ├─┬ combined-stream@1.0.5
│ │ │ └── delayed-stream@1.0.0
│ │ ├── extend@3.0.0
│ │ ├── forever-agent@0.6.1
│ │ ├─┬ form-data@1.0.0-rc2
│ │ │ └── async@1.4.0
│ │ ├─┬ har-validator@1.8.0
│ │ │ ├── bluebird@2.9.34
│ │ │ ├─┬ commander@2.8.1
│ │ │ │ └── graceful-readlink@1.0.1
│ │ │ └─┬ is-my-json-valid@2.12.0
│ │ │   ├── generate-function@2.0.0
│ │ │   ├─┬ generate-object-property@1.2.0
│ │ │   │ └── is-property@1.0.2
│ │ │   ├── jsonpointer@1.1.0
│ │ │   └── xtend@4.0.0
│ │ ├─┬ hawk@3.1.0
│ │ │ ├── boom@2.8.0
│ │ │ ├── cryptiles@2.0.4
│ │ │ ├── hoek@2.14.0
│ │ │ └── sntp@1.0.9
│ │ ├─┬ http-signature@0.11.0
│ │ │ ├── asn1@0.1.11
│ │ │ ├── assert-plus@0.1.5
│ │ │ └── ctype@0.5.3
│ │ ├── isstream@0.1.2
│ │ ├── json-stringify-safe@5.0.1
│ │ ├─┬ mime-types@2.1.3
│ │ │ └── mime-db@1.15.0
│ │ ├── node-uuid@1.4.3
│ │ ├── oauth-sign@0.8.0
│ │ ├── qs@4.0.0
│ │ ├── stringstream@0.0.4
│ │ ├── tough-cookie@2.0.0
│ │ └── tunnel-agent@0.4.1
│ └─┬ sass-graph@2.0.0
│   ├── lodash@3.10.0
│   └─┬ yargs@3.15.0
│     ├── camelcase@1.1.0
│     ├─┬ cliui@2.1.0
│     │ ├─┬ center-align@0.1.1
│     │ │ └─┬ align-text@0.1.3
│     │ │   ├── kind-of@2.0.0
│     │ │   ├── longest@1.0.1
│     │ │   └── repeat-string@1.5.2
│     │ ├─┬ right-align@0.1.3
│     │ │ └─┬ align-text@0.1.3
│     │ │   ├── kind-of@2.0.0
│     │ │   ├── longest@1.0.1
│     │ │   └── repeat-string@1.5.2
│     │ └── wordwrap@0.0.2
│     ├── decamelize@1.0.0
│     └── window-size@0.1.2
├─┬ sass-loader@1.0.3
│ └─┬ loader-utils@0.2.11
│   ├── big.js@3.1.3
│   └── json5@0.4.0
├─┬ style-loader@0.12.3
│ └─┬ loader-utils@0.2.11
│   ├── big.js@3.1.3
│   └── json5@0.4.0
├─┬ webpack@1.10.3
│ ├── async@1.4.0
│ ├── clone@1.0.2
│ ├─┬ enhanced-resolve@0.8.6
│ │ └── graceful-fs@3.0.8
│ ├── esprima@1.2.5
│ ├── interpret@0.6.5
│ ├── memory-fs@0.2.0
│ ├─┬ mkdirp@0.5.1
│ │ └── minimist@0.0.8
│ ├─┬ node-libs-browser@0.5.2
│ │ ├── assert@1.3.0
│ │ ├─┬ browserify-zlib@0.1.4
│ │ │ └── pako@0.2.7
│ │ ├─┬ buffer@3.3.1
│ │ │ ├── base64-js@0.0.8
│ │ │ ├── ieee754@1.1.6
│ │ │ └── is-array@1.0.1
│ │ ├─┬ console-browserify@1.1.0
│ │ │ └── date-now@0.1.4
│ │ ├── constants-browserify@0.0.1
│ │ ├─┬ crypto-browserify@3.2.8
│ │ │ ├── pbkdf2-compat@2.0.1
│ │ │ ├── ripemd160@0.2.0
│ │ │ └── sha.js@2.2.6
│ │ ├── domain-browser@1.1.4
│ │ ├── events@1.0.2
│ │ ├─┬ http-browserify@1.7.0
│ │ │ ├── Base64@0.2.1
│ │ │ └── inherits@2.0.1
│ │ ├── https-browserify@0.0.0
│ │ ├── os-browserify@0.1.2
│ │ ├── path-browserify@0.0.0
│ │ ├── process@0.11.1
│ │ ├── punycode@1.3.2
│ │ ├── querystring-es3@0.2.1
│ │ ├─┬ readable-stream@1.1.13
│ │ │ ├── core-util-is@1.0.1
│ │ │ ├── inherits@2.0.1
│ │ │ └── isarray@0.0.1
│ │ ├─┬ stream-browserify@1.0.0
│ │ │ └── inherits@2.0.1
│ │ ├── string_decoder@0.10.31
│ │ ├── timers-browserify@1.4.1
│ │ ├── tty-browserify@0.0.0
│ │ ├─┬ url@0.10.3
│ │ │ └── querystring@0.2.0
│ │ ├─┬ util@0.10.3
│ │ │ └── inherits@2.0.1
│ │ └─┬ vm-browserify@0.0.4
│ │   └── indexof@0.0.1
│ ├─┬ optimist@0.6.1
│ │ ├── minimist@0.0.10
│ │ └── wordwrap@0.0.3
│ ├─┬ supports-color@3.1.0
│ │ └── has-flag@1.0.0
│ ├── tapable@0.1.9
│ ├─┬ uglify-js@2.4.24
│ │ ├── async@0.2.10
│ │ ├─┬ source-map@0.1.34
│ │ │ └── amdefine@1.0.0
│ │ ├── uglify-to-browserify@1.0.2
│ │ └─┬ yargs@3.5.4
│ │   ├── camelcase@1.1.0
│ │   ├── decamelize@1.0.0
│ │   ├── window-size@0.1.0
│ │   └── wordwrap@0.0.2
│ ├─┬ watchpack@0.2.8
│ │ ├── async@0.9.2
│ │ ├─┬ chokidar@1.0.5
│ │ │ ├─┬ anymatch@1.3.0
│ │ │ │ └─┬ micromatch@2.1.6
│ │ │ │   ├─┬ arr-diff@1.0.1
│ │ │ │   │ └── array-slice@0.2.3
│ │ │ │   ├─┬ braces@1.8.0
│ │ │ │   │ ├─┬ expand-range@1.8.1
│ │ │ │   │ │ └─┬ fill-range@2.2.2
│ │ │ │   │ │   ├── is-number@1.1.2
│ │ │ │   │ │   ├── isobject@1.0.2
│ │ │ │   │ │   ├── randomatic@1.1.0
│ │ │ │   │ │   └── repeat-string@1.5.2
│ │ │ │   │ ├── preserve@0.2.0
│ │ │ │   │ └── repeat-element@1.1.2
│ │ │ │   ├─┬ debug@2.2.0
│ │ │ │   │ └── ms@0.7.1
│ │ │ │   ├── expand-brackets@0.1.2
│ │ │ │   ├── filename-regex@2.0.0
│ │ │ │   ├── kind-of@1.1.0
│ │ │ │   ├─┬ object.omit@0.2.1
│ │ │ │   │ ├─┬ for-own@0.1.3
│ │ │ │   │ │ └── for-in@0.1.4
│ │ │ │   │ └── isobject@0.2.0
│ │ │ │   ├─┬ parse-glob@3.0.2
│ │ │ │   │ ├── glob-base@0.2.0
│ │ │ │   │ ├── is-dotfile@1.0.1
│ │ │ │   │ └── is-extglob@1.0.0
│ │ │ │   └─┬ regex-cache@0.4.2
│ │ │ │     ├── is-equal-shallow@0.1.3
│ │ │ │     └── is-primitive@2.0.0
│ │ │ ├── arrify@1.0.0
│ │ │ ├── async-each@0.1.6
│ │ │ ├─┬ fsevents@0.3.6
│ │ │ │ └── nan@1.8.4
│ │ │ ├── glob-parent@1.2.0
│ │ │ ├─┬ is-binary-path@1.0.1
│ │ │ │ └── binary-extensions@1.3.1
│ │ │ ├── is-glob@1.1.3
│ │ │ ├── path-is-absolute@1.0.0
│ │ │ └─┬ readdirp@1.4.0
│ │ │   ├── graceful-fs@4.1.2
│ │ │   ├─┬ minimatch@0.2.14
│ │ │   │ ├── lru-cache@2.6.5
│ │ │   │ └── sigmund@1.0.1
│ │ │   └─┬ readable-stream@1.0.33
│ │ │     ├── core-util-is@1.0.1
│ │ │     ├── inherits@2.0.1
│ │ │     ├── isarray@0.0.1
│ │ │     └── string_decoder@0.10.31
│ │ └── graceful-fs@3.0.8
│ └─┬ webpack-core@0.6.6
│   ├── source-list-map@0.1.5
│   └─┬ source-map@0.4.4
│     └── amdefine@1.0.0
└─┬ webpack-dev-server@1.10.1
  ├── connect-history-api-fallback@1.1.0
  ├─┬ express@4.13.1
  │ ├─┬ accepts@1.2.11
  │ │ ├─┬ mime-types@2.1.3
  │ │ │ └── mime-db@1.15.0
  │ │ └── negotiator@0.5.3
  │ ├── array-flatten@1.1.0
  │ ├── content-disposition@0.5.0
  │ ├── content-type@1.0.1
  │ ├── cookie@0.1.3
  │ ├── cookie-signature@1.0.6
  │ ├─┬ debug@2.2.0
  │ │ └── ms@0.7.1
  │ ├── depd@1.0.1
  │ ├── escape-html@1.0.2
  │ ├── etag@1.7.0
  │ ├─┬ finalhandler@0.4.0
  │ │ └── unpipe@1.0.0
  │ ├── fresh@0.3.0
  │ ├── merge-descriptors@1.0.0
  │ ├── methods@1.1.1
  │ ├─┬ on-finished@2.3.0
  │ │ └── ee-first@1.1.1
  │ ├── parseurl@1.3.0
  │ ├── path-to-regexp@0.1.6
  │ ├─┬ proxy-addr@1.0.8
  │ │ ├── forwarded@0.1.0
  │ │ └── ipaddr.js@1.0.1
  │ ├── qs@4.0.0
  │ ├── range-parser@1.0.2
  │ ├─┬ send@0.13.0
  │ │ ├── destroy@1.0.3
  │ │ ├─┬ http-errors@1.3.1
  │ │ │ └── inherits@2.0.1
  │ │ ├── mime@1.3.4
  │ │ ├── ms@0.7.1
  │ │ └── statuses@1.2.1
  │ ├── serve-static@1.10.0
  │ ├─┬ type-is@1.6.5
  │ │ ├── media-typer@0.3.0
  │ │ └─┬ mime-types@2.1.3
  │ │   └── mime-db@1.15.0
  │ ├── utils-merge@1.0.0
  │ └── vary@1.0.1
  ├─┬ http-proxy@1.11.1
  │ ├── eventemitter3@1.1.1
  │ └── requires-port@0.0.1
  ├─┬ optimist@0.6.1
  │ ├── minimist@0.0.10
  │ └── wordwrap@0.0.3
  ├─┬ serve-index@1.7.1
  │ ├─┬ accepts@1.2.11
  │ │ └── negotiator@0.5.3
  │ ├── batch@0.5.2
  │ ├─┬ debug@2.2.0
  │ │ └── ms@0.7.1
  │ ├── escape-html@1.0.2
  │ ├─┬ http-errors@1.3.1
  │ │ ├── inherits@2.0.1
  │ │ └── statuses@1.2.1
  │ ├─┬ mime-types@2.1.3
  │ │ └── mime-db@1.15.0
  │ └── parseurl@1.3.0
  ├─┬ socket.io@1.3.6
  │ ├─┬ debug@2.1.0
  │ │ └── ms@0.6.2
  │ ├─┬ engine.io@1.5.2
  │ │ ├── base64id@0.1.0
  │ │ ├─┬ debug@1.0.3
  │ │ │ └── ms@0.6.2
  │ │ ├─┬ engine.io-parser@1.2.1
  │ │ │ ├── after@0.8.1
  │ │ │ ├── arraybuffer.slice@0.0.6
  │ │ │ ├── base64-arraybuffer@0.1.2
  │ │ │ ├── blob@0.0.2
  │ │ │ ├─┬ has-binary@0.1.5
  │ │ │ │ └── isarray@0.0.1
  │ │ │ └── utf8@2.0.0
  │ │ └─┬ ws@0.7.2
  │ │   ├─┬ bufferutil@1.1.0
  │ │   │ ├── bindings@1.2.1
  │ │   │ └── nan@1.8.4
  │ │   ├── options@0.0.6
  │ │   ├── ultron@1.0.2
  │ │   └─┬ utf-8-validate@1.1.0
  │ │     ├── bindings@1.2.1
  │ │     └── nan@1.8.4
  │ ├─┬ has-binary-data@0.1.3
  │ │ └── isarray@0.0.1
  │ ├─┬ socket.io-adapter@0.3.1
  │ │ ├─┬ debug@1.0.2
  │ │ │ └── ms@0.6.2
  │ │ ├── object-keys@1.0.1
  │ │ └─┬ socket.io-parser@2.2.2
  │ │   ├── benchmark@1.0.0
  │ │   ├── component-emitter@1.1.2
  │ │   ├── debug@0.7.4
  │ │   ├── isarray@0.0.1
  │ │   └── json3@3.2.6
  │ └─┬ socket.io-parser@2.2.4
  │   ├── benchmark@1.0.0
  │   ├── component-emitter@1.1.2
  │   ├── debug@0.7.4
  │   ├── isarray@0.0.1
  │   └── json3@3.2.6
  ├─┬ socket.io-client@1.3.6
  │ ├── backo2@1.0.2
  │ ├── component-bind@1.0.0
  │ ├── component-emitter@1.1.2
  │ ├── debug@0.7.4
  │ ├─┬ engine.io-client@1.5.2
  │ │ ├── component-inherit@0.0.3
  │ │ ├─┬ debug@1.0.4
  │ │ │ └── ms@0.6.2
  │ │ ├─┬ engine.io-parser@1.2.1
  │ │ │ ├── after@0.8.1
  │ │ │ ├── arraybuffer.slice@0.0.6
  │ │ │ ├── base64-arraybuffer@0.1.2
  │ │ │ ├── blob@0.0.2
  │ │ │ ├─┬ has-binary@0.1.5
  │ │ │ │ └── isarray@0.0.1
  │ │ │ └── utf8@2.0.0
  │ │ ├─┬ has-cors@1.0.3
  │ │ │ └── global@2.0.1
  │ │ ├─┬ parsejson@0.0.1
  │ │ │ └─┬ better-assert@1.0.2
  │ │ │   └── callsite@1.0.0
  │ │ ├─┬ parseqs@0.0.2
  │ │ │ └─┬ better-assert@1.0.2
  │ │ │   └── callsite@1.0.0
  │ │ ├─┬ parseuri@0.0.4
  │ │ │ └─┬ better-assert@1.0.2
  │ │ │   └── callsite@1.0.0
  │ │ ├─┬ ws@0.7.2
  │ │ │ ├─┬ bufferutil@1.1.0
  │ │ │ │ ├── bindings@1.2.1
  │ │ │ │ └── nan@1.8.4
  │ │ │ ├── options@0.0.6
  │ │ │ ├── ultron@1.0.2
  │ │ │ └─┬ utf-8-validate@1.1.0
  │ │ │   ├── bindings@1.2.1
  │ │ │   └── nan@1.8.4
  │ │ └── xmlhttprequest@1.5.0
  │ ├─┬ has-binary@0.1.6
  │ │ └── isarray@0.0.1
  │ ├── indexof@0.0.1
  │ ├── object-component@0.0.3
  │ ├─┬ parseuri@0.0.2
  │ │ └─┬ better-assert@1.0.2
  │ │   └── callsite@1.0.0
  │ ├─┬ socket.io-parser@2.2.4
  │ │ ├── benchmark@1.0.0
  │ │ ├── isarray@0.0.1
  │ │ └── json3@3.2.6
  │ └── to-array@0.1.3
  ├── stream-cache@0.0.2
  ├─┬ strip-ansi@2.0.1
  │ └── ansi-regex@1.1.1
  ├── supports-color@1.3.1
  └─┬ webpack-dev-middleware@1.2.0
    ├── memory-fs@0.2.0
    └── mime@1.3.4
```

**Comment by fongandrew** [comment]
Did you try updating the SCSS file? The initial build works fine for me, but the watch crashes when the SCSS file is filed.

**Comment by s-panferov** [comment]
@fongandrew fixed in v0.9.0.
